cime/src/drivers/mct/cime_config/config_component_e3sm.xml

```diff
@@ -781,6 +781,40 @@
     <desc>glc2ocn state mapping file decomp type</desc>
   </entry>
 
+  <entry id="GLC2ICE_FMAPNAME">
```

Comment:
@JeremyFyke , @rljacob, @jonbob ,  Jeremy's original changes had these added to config_component.xml.  What is the difference between config_component.xml and config_component_e3sm.xml and which file should these changes be added to?

Reply:
Its possible that config_component_e3sm.xml didn't exist when Jeremy did this work.
The difference is the *e3sm* one is for driver config items that are either unique to E3SM or require E3SM-specific defaults.

Reply:
I don't recall a config_component_e3sm.xml so I think Rob is correct here.